core/trino-main/src/main/java/io/trino/sql/planner/plan/TableWriterNode.java

```diff
@@ -208,28 +209,25 @@ public PlanNode replaceChildren(List<PlanNode> newChildren)
     {
         @Override
         public abstract String toString();
+
+        public abstract CatalogName getCatalogName();
     }
 
     // only used during planning -- will not be serialized
     public static class CreateReference
             extends WriterTarget
     {
-        private final String catalog;
         private final ConnectorTableMetadata tableMetadata;
         private final Optional<NewTableLayout> layout;
+        private final CatalogName catalogName;
```

Comment:
nit: keep the order of fields and constructor parameters the same (we are trying to do this across the codebase)

Reply:
done